src/org/usfirst/frc/team2335/robot/commands/Strafe.java

```diff
@@ -0,0 +1,39 @@
+package org.usfirst.frc.team2335.robot.commands;
+
+import org.usfirst.frc.team2335.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class Strafe extends Command {
+
+    public Strafe() {
+       requires(Robot.driveTrain);
+    }
+
+    // Called just before this Command runs the first time
+    protected void initialize() {
+    }
+
+    // Called repeatedly when this Command is scheduled to run
+    protected void execute() {
+    	Robot.driveTrain.strafe(Robot.oi.getAxis(Robot.STRAFE, 0.8));
+    }
+
+    // Make this return true when this Command no longer needs to run execute()
+    protected boolean isFinished() {
+        return false;
+    }
+
+    // Called once after isFinished returns true
+    protected void end() {
+    	Robot.driveTrain.stopStrafe();
+    }
+
+    // Called when another command which requires one or more of the same
+    // subsystems is scheduled to run
+    protected void interrupted() {
+    }
+}
```

Comment:
So this file has a lot of readability issues. First, the spacing, second the comments.

Any comments you want to keep or change? If you have none then you should remove all or them.